test/run-e2e-ocp.sh

```diff
@@ -123,9 +122,9 @@ main() {
 	wait_for_operators_ready "$OPERATORS_NS"
 
 	local -i ret=0
-	./test/run-e2e.sh --no-deploy --ns "$OPERATORS_NS" || ret=1
-	delete_obo
+	./test/run-e2e.sh --no-deploy --ns "$OPERATORS_NS" || ret=$?
```

Comment:
suppose 
`./test/run-e2e.sh --no-deploy --ns "$OPERATORS_NS" || ret=1`
and
`./test/run-e2e.sh --no-deploy --ns "$OPERATORS_NS" || ret=$?`
have the same effect

Reply:
no, the newer version gives you the exact return code instead of `1` all the time.